src/CMakeLists.txt

```diff
@@ -477,13 +499,15 @@ set(mon_common_files
   auth/cephx/CephxSessionHandler.cc
   erasure-code/ErasureCodePlugin.cc)
 add_library(mon_common_objs OBJECT ${mon_common_files})
+if(${LINUX})
```

Comment:
They were giving me a hard time. But now I end up putting them back in.
Just needed to add safe_io during linking.
So I guess this is going to go in the end.

It got excluded because it is for mountcephfs, which will require major plumbing to get working.
And I think the block with common files is needed for other things as well (if only for unittest_safe_io. So I suspect that the files there are not really mountcephfs only??